contrib/nload/template.py

```diff
@@ -0,0 +1,15 @@
+pkgname = "nload"
+pkgver = "0.7.4"
+pkgrel = 0
+build_style = "gnu_configure"
+make_dir = "."
+hostmakedepends = ["autoconf", "automake"]
```

Comment:
the autoconf is redundant, automake already pulls it

Reply:
fixed now